internal/policies/dconf/dconf.go

```diff
@@ -95,12 +95,27 @@ func (m *Manager) ApplyPolicy(ctx context.Context, objectName string, isComputer
 	dbsPath := filepath.Join(dconfDir, "db")
 	dbPath := filepath.Join(dbsPath, objectName+".d")
 
-	if !isComputer {
+	if !isComputer && len(entries) > 0 {
```

Comment:
Should we have a test with user len(entries) == 0 and no computer data to ensure we don’t fail?

Or is it `"User empty state": {entries: []entry.Entry{}, existingDconfDir: "-"},` (which could be renamed "User empty state without machine policy")?